QATCH/QModel/src/models/static_v4_fusion/v4_fusion_dataprocessor.py

```
@@ -69,298 +69,256 @@ def load_content(data_dir: str, num_datasets: Union[int, float] = np.inf) -> Lis
         df["Resonance_Frequency"] = -(df["Resonance_Frequency"] - base_rf)
 
+        return df, base_d, base_rf
+
+    @staticmethod
+    def get_reg_features(df: pd.DataFrame) -> pd.DataFrame:
+        """Generate features for regression tasks."""
+        df = df.copy()
+        df = df.drop(columns=FusionDataprocessor.DROP)
+
+        # Apply baseline correction (time-aware)
+        df, base_d, base_rf = FusionDataprocessor.apply_baseline_correction(
```

Ruff (0.13.1)

Unpacked variable `base_d` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)